utils/index.js

```diff
@@ -29,10 +29,11 @@ exports.getReactScriptsVersion = function getReactScriptsVersion(cliVersion) {
   }
 
   const { dependencies } = packageJson;
+  const reactScriptsVersionString = /^[~^]?([.0-9]+)$/.exec(dependencies['react-scripts'])[1];
   const versions = {
-    major: Number(semver.major(dependencies['react-scripts'])),
-    minor: Number(semver.minor(dependencies['react-scripts'])),
-    patch: Number(semver.patch(dependencies['react-scripts'])),
+    major: Number(semver.major(reactScriptsVersionString)),
```

Comment:
semver.major only accepts a version string, not a ^ or ~ prefixed comparison string.

Reply:
Hum, did you get an error with the previous behavior? I'd rather prefer that the prefix is taken into account because having `^1.3.0` does not mean that you have `1.3.0` installed hence you could have `1.3.7`. That's the reason I use the `semver` utils for.

Reply:
Yes:

TypeError: Invalid Version: ^2.1.3
    at new SemVer (/Users/johnsingleton/cra-test/my-app/node_modules/semver/semver.js:312:11)

I think because "^2.1.3" is a constraint not a version number. As is the code is just pulling whatever is in package.json, so I don't think it's getting the actual installed version anyway.

Reply:
Yeah you are right, there could always be a difference with the actual version installed but at least I feel we are a bit closer to the truth. 😄  I'll need to look a bit more in `semver` docs or look for another solution before I settle for this. If we don't have another solution then we'll roll with it.

Reply:
I've found a better way to get the package version and to be sure it is the actual version installed by the user. However I don't think it should be a part of this PR. I'll make the development in another PR. I'll just change your PR destination to `develop` and in the meantime can you delete the modifications you did to the `getReactScriptsVersion` function, please?